launch/tier4_control_launch/launch/control.launch.py

```diff
@@ -174,11 +176,6 @@ def launch_setup(context, *args, **kwargs):
         parameters=[
             vehicle_cmd_gate_param,
             vehicle_info_param,
-            {
-                "use_emergency_handling": LaunchConfiguration("use_emergency_handling"),
-                "use_external_emergency_stop": LaunchConfiguration("use_external_emergency_stop"),
-                "use_start_request": LaunchConfiguration("use_start_request"),
-            },
```

Comment:
@isamu-takagi Do you wanna designate these variables not by a yaml file but by ros2 launch command?

Reply:
I don't think it will be used from a command, so a file is OK.

Reply:
Thanks